python/src/linalg.cpp

```diff
@@ -241,4 +241,28 @@ void init_linalg(py::module_& parent_module) {
             tuple(array, array, array): The ``U``, ``S``, and ``Vt`` matrices, such that
               ``A = U @ diag(S) @ Vt``
       )pbdoc");
+  m.def(
+      "inv",
+      &inv,
+      "a"_a,
+      py::kw_only(),
+      "stream"_a = none,
+      R"pbdoc(
+        inv(a: array, *, stream: Union[None, Stream, Device] = None) -> array
+
+        Compute the (multiplicative) inverse of a (square) matrix.
```

Comment:
```suggestion
        Compute the inverse of a square matrix.
```

Maybe just me but the multiplicative seems quite clear.. you could add it back if you disagree.